apps/src/userHeaderStatsigReporter/userHeaderStatsigReporter.js

```diff
@@ -1,6 +1,18 @@
 import statsigReporter from '../lib/util/StatsigReporter';
 
-document.addEventListener('DOMContentLoaded', function () {
+if (document.readyState !== 'loading') {
+  console.log('Document is ready');
```

Comment:
Yes, if we see this particular log output, it should serve as confirmation that we are making the correct fix.